BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -1004,6 +1003,8 @@
 "Dismiss" = "Dismiss";
 "SaveAndProtectYourData" = "Save and protect your data";
 "TheVaultProtectsMoreThanJustPasswordsStoreSecureLoginsIdsCardsAndNotesSecurelyHere" = "The vault protects more than just your passwords. Store secure logins, IDs, cards and notes securely here.";
+"SendSensitiveInformationSafetly" = "Send sensitive information, safetly";
```

Comment:
🤔 Maybe I'm missing this spelling since I'm not a native English speaker but I believe this should be `safely`, right? Just in case I looked it up => [Source](https://whichiscorrect.com/safely-or-safetly/)
```suggestion
"SendSensitiveInformationSafely" = "Send sensitive information, safely";
```

Reply:
ah okay, yeah good catch.  ill make a quick PR right now to fix this since I just merged.  sorry federico!